internal/pkg/template/templates/app/app.yml

```diff
@@ -42,6 +42,9 @@ Resources:
               Service: cloudformation.amazonaws.com
             Action:
               - sts:AssumeRole
+{{- if .PermissionsBoundary }}
+      PermissionsBoundary: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}'
```

Comment:
Just to make sure - we are assuming that the PermissionBoundary lives in the account that deploys the app, is that right?

Reply:
For these IAM roles, yes. If they deploy cross-account environments, we'll refer to those account IDs.

Reply:
does that mean they'd need a copy of the permission boundary in every account they deploy in to? would it make sense to let them specify the full ARN and reference a permission boundary in a specific account? (i have no idea if that's possible, just checking)

Reply:
They'd duplicate the same managed policy in all the accounts! The policy name should be the same in the app and env accounts as well (that's why we can safely use the flag value for _all_ our iam roles)

Reply:
We might also need to upgrade the app template version as well 👀

Reply:
upgrade app template version. thanks!